tests/python/gpu/test_operator_gpu.py

```diff
@@ -1657,7 +1657,14 @@ def test_cross_device_autograd():
 
     assert_almost_equal(dx, x.grad.asnumpy())
 
+@unittest.skip('''
```

Comment:
Please link the jira issue there and remove this description (it shows up in the test results)

Reply:
Fixed it. Thank you!